src/Servers/Kestrel/Core/src/Internal/Http3/Http3StreamContext.cs

```diff
@@ -6,11 +6,33 @@
 using Microsoft.AspNetCore.Connections;
 using Microsoft.AspNetCore.Http.Features;
 using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.WebTransport;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3;
 
 internal sealed class Http3StreamContext : HttpConnectionContext
 {
+    public Http3StreamContext(
```

Comment:
Do we still need the old constructor? It looks like you're getting everything from the connection now.

Reply:
Ok I wasn't sure if removing a constructor is a problem. I just found that a lot of the properties where available from the Connection and I was going to add more. I will remove the old one then if it is ok.